task02/task02/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task02
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите первую сторону прямоугольника");
+            var a = double.Parse(Console.ReadLine());
+
+            Console.WriteLine("Введите вторую сторону прямоугольника");
+            var b = double.Parse(Console.ReadLine());
+
+            var area = a * b;
+            var perimeter = (a + b) * 2;
+
+            var double_a = Math.Pow(a, 2);
+            var double_b = Math.Pow(b, 2);
+            var diagonal = Math.Sqrt(double_a+double_b);
```

Comment:
Можно проще. Зачем лишние локальные переменные?
```suggestion
            var diagonal = Math.Sqrt(a * a + b * b);
```